tests/CMakeLists.txt

```diff
@@ -1,71 +1,70 @@
 include(GoogleTest)
-pkg_check_modules(CODEC2 REQUIRED codec2)
 
 include_directories (
     ${TEST_SOURCE_DIR}
     ..
 )
 
 add_executable (ConvolutionTest ConvolutionTest.cpp)
-target_link_libraries(ConvolutionTest m17cxx gtest pthread)
+target_link_libraries(ConvolutionTest m17cxx GTest::GTest)
```

Comment:
please refer to https://github.com/mobilinkd/m17-cxx-demod/pull/4 for the presence of pthread. It may not be necessary on all systems, but it will probably break some if removed.

Reply:
Ok - I can see pthread would be an issue.  Windows definitely does not want it in there.  I've put a conditional variable in the CMakeLists.txt that is set when not WIN32.